src/primitives.es6

```diff
@@ -33,7 +42,7 @@ export class ElementType {
   toRefract(options = {}) {
     const attributes = this.convertAttributesToRefract('toRefract');
     const initial = {
-      element: this.element,
+      element: this.elementName(),
```

Comment:
I would actually like to provide some method for `content`. The `meta` property is also something special, as it's a `Meta` class that has Minim instances for each key. I'm going to think about this more for now, but this is a good spot.